lingpy3/algorithm/cython/_calign.py

```diff
@@ -165,7 +165,7 @@ def globalign(
             almB += [seqB[i-1]]
             i -= 1
             j -= 1
-        else:
+        else:  # pragma: no cover
```

Comment:
ah, I see. But if it's not covered, I should probably still look into it. It may hide a real problem, or it just requires one more string in our test data, with a slightly different structure (or turning alignments a, b to b, a, as this is symmetric).